app/assets/hee/blocks/content/footer/hee-anchorlinks-sticky/anchorlinks-sticky.js

```diff
@@ -71,7 +72,10 @@ export default () => {
      * @returns void
      */
     toggleStickyToolbar() {
-      if (!this.isElementInViewport(this.sidebarAnchorLinks.querySelector('h2'))) {
+      const tocBlockVisible = this.isElementInViewport(this.sidebarAnchorLinks.querySelector('h2'));
+      const footerVisible =   this.footer? this.isElementInViewport(this.footer): false;
```

Comment:
@FranciscoRuizHEE - Wondering if we can simplify this as `const footerVisible = this.isElementInViewport(this.footer);` as we always will have a footer on all pages ?